### PR TITLE
Disable /ZI and incremental linking

### PR DESCRIPTION
**Pull request by Alan-Jowett**
Disable /ZI and incremental linking

Resolves: #545 

Signed-off-by: Alan Jowett <alanjo@microsoft.com>

### CODE REVIEW

**Review by shankarseal — APPROVED** [review]
Thank you!

File: tools/dnsflood/dnsflood.vcxproj [hunk 1]
```diff
@@ -81,7 +81,7 @@
     <LinkIncremental>false</LinkIncremental>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
-    <LinkIncremental>true</LinkIncremental>
+    <LinkIncremental>false</LinkIncremental>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
     <LinkIncremental>false</LinkIncremental>
```
Comment (dthaler) [inline tools/dnsflood/dnsflood.vcxproj:87]: Why is it different for debug vs release?

File: tests/libs/common/common_tests.vcxproj [hunk 2]
```diff
@@ -128,6 +128,7 @@
       <TreatWarningAsError>true</TreatWarningAsError>
       <RuntimeLibrary>MultiThreadedDebugDll</RuntimeLibrary>
       <DebugInformationFormat>ProgramDatabase</DebugInformationFormat>
+      <SupportJustMyCode>false</SupportJustMyCode>
```
Comment (dthaler) [inline tests/libs/common/common_tests.vcxproj:131]: LinkIncremental is still true in lines 79 and 85

File: tests/client/ebpf_client.vcxproj [hunk 3]
```diff
@@ -83,7 +83,7 @@
     <LinkIncremental>false</LinkIncremental>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
-    <LinkIncremental>true</LinkIncremental>
+    <LinkIncremental>false</LinkIncremental>
```
Comment (dthaler) [inline tests/client/ebpf_client.vcxproj:86]: What about line 80?

**Review by dthaler — CHANGES_REQUESTED** [review]
There are still a number of <LinkIncremental>true</LinkIncremental> across the repo not touched by this PR yet.  For example tools\elfhelper\elfhelper.vcxproj

I'd recommend doing:
```
findstr /s LinkIncremental.true *.vcxproj
```
And verifying that everything shown is intentional.

**Comment by Alan-Jowett** [comment]
Switched everything within this repo to disable incremental linking. We need a PR in the ebpf-verifier repo to disable it there if we need to.

**Review by shankarseal — APPROVED** [review]

**Review by dthaler — APPROVED** [review]

**Pull request merged**